### PR TITLE
clean data which is not consistency in different cfs

### PR DESCRIPTION
**Pull request by wentaoxu**
as title says

### CODE REVIEW

File: src/server/debug.rs [hunk 1]
```diff
@@ -341,6 +370,90 @@ impl Debugger {
...
+        let write_handle = box_try!(get_cf_handle(&self.db, CF_WRITE));
+        let default_handle = box_try!(get_cf_handle(&self.db, CF_DEFAULT));
+
+        if iter.valid() {
+            loop {
+                let key = iter.key().to_vec();
+                let value = iter.value().to_vec();
+                let write = box_try!(Write::parse(&value));
+                if write.short_value == None {
+                    if let Ok(None) = self.db.get_cf(default_handle, key.as_ref()) {
```
Comment (siddontang) [inline src/server/debug.rs]: seem default and write have the different key format?
Write contains the commit ts, and data contains the start ts, so we can get the data with write format key?

Comment (zhangjinpeng1987) [reply src/server/debug.rs]: ooooops

Comment (zhangjinpeng1987) [reply src/server/debug.rs]: Should truncate commit ts at first,  and then append start ts.

Comment (wentaoxu) [reply src/server/debug.rs]: yes, it is a big mistake, I think I have deleted keys which should be deleted and should not! Now default cf need to verify too.

File: src/server/debug.rs [hunk 2]
```diff
@@ -341,6 +370,90 @@ impl Debugger {
...
+    db: Arc<DB>,
+    lock_iter: DBIterator,
+    //    default_iter: DBIterator,
+    write_iter: DBIterator,
+}
+
+impl RegionVerifier {
+    fn new(db: Arc<DB>, reg: Region) -> Result<Self> {
+        let region = reg.clone();
+        let start_key = keys::data_key(region.get_start_key());
```
[the remark this answers is not in the document]
Comment (disksing) [reply src/server/debug.rs]: No need it. The region is loaded from pd-server.

**Pull request closed without merging**